packages/opentelemetry-sdk-trace-base/test/common/BasicTracerProvider.test.ts

```diff
@@ -42,6 +42,7 @@ import {
   AlwaysOnSampler,
   AlwaysOffSampler,
 } from '../../src';
+import { env } from 'process';
```

Comment:
Nit: typically we use the global `process.env` rather than importing from `'process'`.

Reply:
This is fixed. I didn't see that [the right process.env was already defined here](https://github.com/open-telemetry/opentelemetry-js/blob/947b3073ed6723be1b202ec76c6784fa7b0acd36/packages/opentelemetry-sdk-trace-base/test/common/BasicTracerProvider.test.ts#L64-L68) so I just used `envSource` defined there instead of importing `process.env` again.